Main.java

```diff
@@ -1,78 +1,195 @@
+    case 'T':    
+	  System.out.println("Calculation for a Trapezoid");
+	  System.out.println("Input first side");
+	  float a = scan.nextFloat();
+	  System.out.println("Input second side");
+	  float b = scan.nextFloat();
+	  System.out.println("Input third side");
+	  float c = scan.nextFloat();
+	  System.out.println("Input fourth side");
+	  float d = scan.nextFloat();
+	  System.out.println("Input height");
+	  Float ht = scan.nextFloat();
+		      
+	  Trapezoid t = new Trapezoid(a, b, c, d,ht);
+	  System.out.printf("The perimeter of the Trapezoid with First Side: %.2f, Second Side: %.2f, Third Side: %.2f, and Fourth Side: %.2f is %.2f", a, b, c, d, t.calculatePerimeter()); 
+	  System.out.println(" ");
+	  System.out.printf("The area of the Trapezoid with First Side: %.2f, Second Side: %.2f, Third Side: %.2f, and Fourth Side: %.2f and hheight of: %.2f is %.2f", a, b, c, d, ht, t.calculateArea()); 
+	  break;
+	
+         
+         
+          //Rhombus case
+    case 'H':
+          System.out.println("This programs calculates the area and perimeter of the Rhombus");
+          System.out.println("Input the first diagonal");
+          float d1 = scan.nextFloat();
+          System.out.println("Input second diagonal");
+          float d2 = scan.nextFloat();
+         System.out.println("Input length of side");
+          float rs = scan.nextFloat();
+          
+          Rhombus Rho = new Rhombus(d1, d2, rs);
+          System.out.printf("The area of the Rhombus with First diagonal: %.2f and Second diagonal: %.2f is %.2f", d1, d2, Rho.area());
+          System.out.println(" ");
+          System.out.printf("The perimeter of the Rhombus with First diagonal: %.2f, Second diagonal: %.2f  and side length: %.2f is %.2f", d1, d2,rs, Rho.perimeter());
+          break;
     
-    Quadilateral q = p;
+          //Square case
+    case 'S':
+    	 System.out.println("This programs calculates the area and perimeter of the Square");
+    	 System.out.println("Input side length");
+    	 float length = scan.nextFloat();
+    	 
+    	 Square sqr = new Square(length);
+    	 System.out.printf("The area of a square with side: %.2f is %.2f",length, sqr.area());
+    	 System.out.println(" ");
+    	 System.out.printf("The perimeter of a square with side: %.2f is %.2f",length, sqr.perimeter());
+    	 break;
+    	 
+       //Rectangle case
+    case 'R':
+    	System.out.println("This programs calculates the area and perimeter of the Rectangle");
+    	System.out.println("Input length");
+    	float l1 = scan.nextFloat();
+    	System.out.println("Insert breadth");
+    	float b2 = scan.nextFloat();
+    	
+    	Rectangle rec = new Rectangle(l1,b2);
+   	 	System.out.printf("The area of a rectangle with length: %.2f and breadth: %.2f is %.2f",l1,b2,rec.area());
+   	 	System.out.println("");
+   	 	System.out.printf("The perimeter of a rectangle with length: %.2f and breadth: %.2f is %.2f", l1,b2,rec.perimeter());
+      break;
+
+
```

Comment:
'default' option is not there, which should come after all the cases in case the user enters a wrong input